recipes/odbc2parquet/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "odbc2parquet" %}
+{% set version = "2.0.2" %}
+{% set build_num = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pacman82/odbc2parquet/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 8d635c8407d7e79816f9c2cc102f0cff28876e2ea95f47f82b8e108c534b0846
+
+build:
+  number: {{ build_num }}
+  script:
+    - cargo install --root "$PREFIX" --path .
+    - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+
+requirements:
+  build:
+    - {{ compiler('c') }}       # [linux]
+    - {{ compiler('rust') }}
+    - unixodbc                  # [unix]
```

Comment:
This should not be necessary. Do you get an error if you remove it?

Reply:
Yes

Reply:
Can you post that? I'm unable to navigate to it on mobile.

Reply:
```
error: linking with `/home/conda/staged-recipes/build_artifacts/odbc2parquet_1694870537904/_build_env/bin/x86_64-conda-linux-gnu-cc` failed: exit status: 1
  |
  = note: LC_ALL="C" PATH="/home/conda/staged-recipes/build_artifacts/odbc2parquet_1694870537904/_build_env/lib/rustlib/x86_64-unknown-linux-gnu/bin:/home/conda/staged-recipes/build_artifacts/odbc2parquet_1694870537904/_build_env/.cargo/bin:/home/conda/staged-recipes/build_artifacts/odbc2parquet_1694870537904/_build_env/bin:/home/conda/staged-recipes/build_artifacts/odbc2parquet_1694870537904/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehol/bin:/opt/conda/condabin:/home/conda/staged-recipes/build_artifacts/odbc2parquet_1694870537904/_build_env/bin:/home/conda/staged-recipes/build_artifacts/odbc2parquet_1694870537904/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehol/bin:/opt/conda/bin:/opt/conda/condabin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/home/conda/bin" VSLANG="1033" "/home/conda/staged-recipes/build_artifacts/odbc2parquet_1694870537904/_build_env/bin/x86_64-conda-linux-gnu-cc" "-m64" "/tmp/rustc3qkB0L/symbols.o" "/home/conda/staged-recipes/build_artifacts/odbc2parquet_1694870537904/work/target/x86_64-unknown-linux-gnu/release/deps/odbc2parquet-0b6faf9750dd8881.odbc2parquet.8e1cb2c0a8aea5b0-cgu.01.rcgu.o" "-Wl,--as-needed" "-L" "/home/conda/staged-recipes/build_artifacts/odbc2parquet_1694870537904/work/target/x86_64-unknown-linux-gnu/release/deps" "-L" "/home/conda/staged-recipes/build_artifacts/odbc2parquet_1694870537904/work/target/release/deps" "-L" "/home/conda/staged-recipes/build_artifacts/odbc2parquet_1694870537904/work/target/x86_64-unknown-linux-gnu/release/build/lz4-sys-a27774d65445ae18/out" "-L" "/home/conda/staged-recipes/build_artifacts/odbc2parquet_1694870537904/work/target/x86_64-unknown-linux-gnu/release/build/zstd-sys-a260d1a5697fa5b2/out" "-L" "/home/conda/staged-recipes/build_artifacts/odbc2parquet_1694870537904/_build_env/lib/rustlib/x86_64-unknown-linux-gnu/lib" "-Wl,-Bstatic" "/tmp/rustc3qkB0L/libzstd_sys-1cb384104906291f.rlib" "/tmp/rustc3qkB0L/liblz4_sys-3ca9b5e41b9d9f44.rlib" "/home/conda/staged-recipes/build_artifacts/odbc2parquet_1694870537904/_build_env/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcompiler_builtins-b114db70ea0690b1.rlib" "-Wl,-Bdynamic" "-lodbc" "-lgcc_s" "-lutil" "-lrt" "-lpthread" "-lm" "-ldl" "-lc" "-Wl,--eh-frame-hdr" "-Wl,-z,noexecstack" "-L" "/home/conda/staged-recipes/build_artifacts/odbc2parquet_1694870537904/_build_env/lib/rustlib/x86_64-unknown-linux-gnu/lib" "-o" "/home/conda/staged-recipes/build_artifacts/odbc2parquet_1694870537904/work/target/x86_64-unknown-linux-gnu/release/deps/odbc2parquet-0b6faf9750dd8881" "-Wl,--gc-sections" "-pie" "-Wl,-z,relro,-z,now" "-Wl,-O1" "-nodefaultlibs" "-Wl,-rpath-link,/home/conda/staged-recipes/build_artifacts/odbc2parquet_1694870537904/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehol/lib" "-Wl,-rpath,/home/conda/staged-recipes/build_artifacts/odbc2parquet_1694870537904/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehol/lib"
  = note: /home/conda/staged-recipes/build_artifacts/odbc2parquet_1694870537904/_build_env/bin/../lib/gcc/x86_64-conda-linux-gnu/12.3.0/../../../../x86_64-conda-linux-gnu/bin/ld: cannot find -lodbc: No such file or directory
          collect2: error: ld returned 1 exit status
```

Reply:
There is some link flag missing. Can you try the following:

```
export RUSTFLAGS="-Clink-args=-Wl,-L$PREFIX/lib"
```

Reply:
This will otherwise break later on if you cross-compile.